src/main/resources/META-INF/rewrite/ibm-java.yml

```diff
@@ -110,4 +112,30 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.ibm.security.auth.module.Krb5LoginModule
       newFullyQualifiedTypeName: com.sun.security.auth.module.Krb5LoginModule
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJavaXMLWSModuleProvided
+displayName: Do not use `java.xml.ws` module in application
+description: The `java.xml.ws` module was removed in Java11. This recipe removes the module from the application by removing groupId:`javax.xml.ws` and artifactId 'jaxws-api` from both the maven and gradle build files.
```

Comment:
We should mention that the appropriate feature will be provided in their server.xml file